src/adult/adult_specific_block.txt

```diff
@@ -13,3 +13,4 @@
 ||pemersatudotfun.com/ads.js$script
 ||slunkenchukkar.com/nwnJuadwxPHeiD3/52345$script,domain=manhwaid.club
 ||sodasusu.com/banner/
+||wibufile.com/api/beenv^
```

Comment:
is this adv server? seem video host

Reply:
No, I'm confident it is an ad server endpoint.

<details>

<summary>Source code:</summary>

`https://wibufile.com/embed/NexRhZfVcMM3yPS, line 95`

```html
<script type="text/javascript">
   $(document).ready(function() {
      let video = document.querySelector("#video");
      if (video) {
         const player = new Plyr(video);
         player.source = {
            type: 'video',
            title: "BSKh-Cour2-03-480p-SAMEHADAKU.CARE.mp4",
            sources: [{
               src: "https://wibufile.com/videos/s1/pNir8OmpGAmZoRQ_1690044430.mp4",
               type: "video/mp4",
            }],
         };

         axios.get('/api/beenv')
            .then(response => {
               const data = response.data;
               processData(data);
            })
            .catch(error => {
               console.error(error);
            });

         function processData(data) {
            if (data.adstype == "image" && data.adsimg1 !== "" || data.adsimg2 !== "" || data.adsimg3 !== "") {
               const adOverlay = document.querySelector('.ik-overlay');
               const adLink = document.getElementById('ik-link');
               const closeBtn = document.getElementById('close-btn');
               var ads = [];
               if (data.adsimg1 !== "") {
                  const ads1 = {
                     url: data.adsimg1,
                     href: data.adsurl1
                  };
                  ads.push(ads1);
               }
               if (data.adsimg2 !== "") {
                  const ads2 = {
                     url: data.adsimg2,
                     href: data.adsurl2
                  };
                  ads.push(ads2);
               }
               if (data.adsimg3 !== "") {
                  const ads3 = {
                     url: data.adsimg3,
                     href: data.adsurl3
                  };
                  ads.push(ads3);
               }
               var randomAd = ads[Math.floor(Math.random() * ads.length)];
               adLink.href = randomAd.href;
               adLink.target = "_blank";
               adLink.querySelector("img").src = randomAd.url;

               adLink.addEventListener('click', function(event) {
                  event.preventDefault();
                  window.open(adLink.href, '_blank');
               });

               adOverlay.style.display = 'block';

               closeBtn.addEventListener('click', function() {
                  adOverlay.style.display = 'none';
               });
            } else if (data.adstype == "video" && data.adsvid1 !== "" || data.adsvid2 !== "" || data.adsvid3 !== "") {
               const adPlayer = new Plyr('#ik-video', {
                  hideControls: false,
                  controls: ['play', 'progress', 'current-time', 'mute', 'volume'],
                  autoplay: false
               });
               const adPlayerId = document.querySelector('#ik-player');
               const videoWrapper = document.querySelector('.video-wrapper');
               var adsvid = [];
               if (data.adsvid1 !== "") {
                  const adsvid1 = {
                     title: data.adsvidtitle1,
                     src: data.adsvid1,
                     url: data.adsvidurl1
                  };
                  adsvid.push(adsvid1);
               }
               if (data.adsvid2 !== "") {
                  const adsvid2 = {
                     title: data.adsvidtitle2,
                     src: data.adsvid2,
                     url: data.adsvidurl2
                  };
                  adsvid.push(adsvid2);
               }
               if (data.adsvid3 !== "") {
                  const adsvid3 = {
                     title: data.adsvidtitle3,
                     src: data.adsvid3,
                     url: data.adsvidurl3
                  };
                  adsvid.push(adsvid3);
               }
               var randomAdsVid = adsvid[Math.floor(Math.random() * adsvid.length)];
               adPlayer.source = {
                  type: 'video',
                  title: randomAdsVid.title,
                  sources: [{
                     src: randomAdsVid.src,
                     type: 'video/mp4',
                  }],
               };
               var adss = true;
               player.on('play', function() {
                  if (adss) {
                     if (player.fullscreen.active) {
                        player.fullscreen.exit();
                     }
                     player.muted = true;
                     const currentTime = player.currentTime;
                     videoWrapper.style.display = 'none';
                     adPlayerId.style.display = 'block';
                     adPlayer.play();
                     const skipButton = document.querySelector('.plyr__skip-button');
                     skipButton.style.display = 'block';
                     let countdown = 5;
                     const countdownInterval = setInterval(function() {
                        countdown--;
                        skipButton.innerText = 'Please wait (' + countdown + ')';
                        if (countdown <= 0) {
                           skipButton.innerText = 'Skip';
                           skipButton.addEventListener('click', function() {
                              skipButton.style.display = 'none';
                              adPlayer.stop();
                              adPlayerId.style.display = 'none';
                              videoWrapper.style.display = 'block';
                              player.currentTime = currentTime;
                              player.muted = false;
                              adss = false;
                           });
                        }
                     }, 1000);

                     adPlayer.on('play', function() {
                        const adPlayerWrapper = document.querySelector('#ik-wrapper');
                        adPlayerWrapper.addEventListener('click', handleScreenClick);

                        function handleScreenClick() {
                           window.open(randomAdsVid.url, '_blank');
                        }
                     });

                     adPlayer.on('ended', function() {
                        skipButton.style.display = 'none';
                        adPlayer.stop();
                        adPlayerId.style.display = 'none';
                        videoWrapper.style.display = 'block';
                        player.currentTime = currentTime;
                        player.muted = false;
                        adss = false;
                     });
                  }
               });
            }

         }
      }
   });
</script>
```

</details>